ui/app/templates/components/date-dropdown.hbs

```diff
@@ -36,7 +37,12 @@
     <nav class="box menu">
       <ul class="menu-list">
         {{#each this.years as |year|}}
-          <button type="button" class="link" {{on "click" (queue (fn this.selectStartYear year) (action D.actions.close))}}>
+          <button
+            type="button"
+            class="button link"
+            disabled={{if (eq year this.disabledYear) true false}}
+            {{on "click" (queue (fn this.selectStartYear year) (action D.actions.close))}}
```

Comment:
if you don't want to use queue, you can pass D.actions or just D into this.selectStartYear as a param and then call it in the function. e.g. function(D.actions) { code code ... D.actions.close}

Reply:
:point_up:. I think we should avoid using queue when possible.